packages/react/src/components/ComboBox/ComboBox.test.tsx

```diff
@@ -1152,19 +1046,15 @@ describe('ComboBox', () => {
       text: 'ManuallyEnteredValue',
       selected: true,
     };
-    safeMount(
-      <ComboBox multiSelect options={DEFAULT_OPTIONS} defaultSelectedKey={['1', '2', '3']} allowFreeform />,
-      wrapper => {
-        const inputElement = wrapper.find('input');
-        _verifyStateVariables(wrapper, 'none', DEFAULT_OPTIONS, [0, 1, 2]);
```

Comment:
It might be good if you could find some other way to verify the things this was testing (possibly based on the DOM instead of internal state?)

Reply:
Oops missed this before but this `_verifyStateVariables` function was basically testing for stuff like focus status, available options and currently selected options and I basically simplified the tests to only worry about the expected value of the combobox element should be depending on different scenarios like an input text being autoselected onBlur in a freeform combobox.